src/view/pages/gettingStarted.tsx

```diff
@@ -136,6 +136,12 @@ export class GettingStartedPage extends React.Component {
                     {/* prettier-ignore */}
                     <div id="CLUE" className="inv">
                         <h2> Tutorial for CLUE </h2>
+                        <h3> 0. Enable Preview Mode (This is required)</h3>
+                        <p> To use the CLUE, you must check the <strong>"Device Simulator Express: Enable Preview Mode"</strong> setting. To access your settings:</p>
+                        <ul>

```

Comment:
Optional -> in my README PR, I have and image + gif as part of my tutorial to enable preview mode. You could use these if you want to. ^^

Reply:
Will do after README change is merged.